node/src/accountant/mod.rs

```diff
@@ -2606,6 +2640,222 @@ mod tests {
             .scan_started_at_result(None)
             .start_scan_params(&payable_start_scan_params_arc)
             .start_scan_result(Err(StartScanError::NothingToProcess));
+        let (subject, new_payable_expected_computed_interval, receivable_scan_interval) =
+            set_up_subject_for_no_pending_payables_found_startup_test(
+                test_name,
+                &scan_for_pending_payables_notify_later_params_arc,
+                &scan_for_new_payables_notify_later_params_arc,
+                &scan_for_new_payables_notify_params_arc,
+                &scan_for_retry_payables_notify_params_arc,
+                &compute_interval_params_arc,
+                &scan_for_receivables_notify_later_params_arc,
+                config,
+                pending_payable_scanner,
+                receivable_scanner,
+                payable_scanner,
+            );
+        let peer_actors = peer_actors_builder().build();
+        let subject_addr: Addr<Accountant> = subject.start();
+        let subject_subs = Accountant::make_subs_from(&subject_addr);
+        send_bind_message!(subject_subs, peer_actors);
+
+        send_start_message!(subject_subs);
+
+        // The system is stopped by the NotifyLaterHandleMock for the Receivable scanner
+        system.run();
+        let pp_scan_started_at = assert_pending_payable_scanner_for_no_pending_payable_found(
+            test_name,
+            consuming_wallet,
+            pending_payable_start_scan_params_arc,
+            scan_for_pending_payables_notify_later_params_arc,
+        );
+        let p_scheduling_happened_at = assert_payable_scanner_for_no_pending_payable_found(
+            scan_for_new_payables_notify_later_params_arc,
+            new_payable_expected_computed_interval,
+            scan_for_new_payables_notify_params_arc,
+            scan_for_retry_payables_notify_params_arc,
+            compute_interval_params_arc,
+        );
+        assert_receivable_scanner(
+            test_name,
+            earning_wallet,
+            receivable_start_scan_params_arc,
+            receivable_scan_interval,
+            scan_for_receivables_notify_later_params_arc,
+        );
+        // We expect the PendingPayableScanner to take place before the PayableScanner.
+        // I do believe it's impossible for these two events to happen within the same nanosecond,
+        // until somebody fiddles with MASQ tests on a true supercomputer.
+        assert!(
+            pp_scan_started_at < p_scheduling_happened_at,
+            "We failed to prove that the PendingPayableScanner runs before the PayableScanner."
+        );
+    }
+
+    #[test]
+    fn accountant_scans_after_startup_and_detects_straggled_pending_payable() {
```

Comment:
How about if we call it, `accountant_detects_straggling_pending_payable_during_startup`.